README.md

```diff
@@ -1,2 +1,48 @@
+## Feature
+### Obiettivi finali da raggiungere 
+* Costruire un sistema di tasse e benefici completo per l'Italia, definendo ogni singola Variabile, Parametro, Riforma,Entità e Formula;
+* Implementare un software grafico dalle diverse funzionalità che possa essere utilizzato dalla pubblica amministrazione per misurare l'impatto che porterebbe al sistema Italiano il cambiamento di una Variabile/Parametro/Riforma tramite un sistema di simulazioni.
+### Prossimi obiettivi da raggiungere
+L'obiettivo dal quale si vuole partire è quello di definire un sistema per gestire la disoccupazione. In particolare, i prossimi obiettivi da raggiungere per il campo della disoccupazione sono:
+- [ ] Definire variabili, parametri, riforme.... per creare una base dalla quale partire per sviluppare delle features;
+- [ ] __Feature 1__: Gestione delle pensioni di disoccupazione;
+- [ ] __Feature 2__: Gestione di un ipotetico reddito di base;
+- [ ] __Feature 3__: Statistiche sulla disoccupazione (tasso di disoccupazione derivante da inserimento/eliminazione di variabili (Es. imposte per le piccole/medie imprese, occupazione nel settore pubblico, Jobs Act...)
+
+## Prerequisiti per l'installazione
+### Sistemi operativi supportati
+* Distribuzioni Gnu/Linux (in particolare Debian e Ubuntu);
+* Mac OS X ;
+* Windows;
+### Strumenti necessari
+* [Git](https://git-scm.com/)
+* [Python v. 2.7](https://www.python.org/download/releases/2.7/)
+* [Pip](https://pip.pypa.io/en/stable/installing/)
+* [Numpy](https://pypi.python.org/pypi/numpy)
+* [Pew](https://pypi.python.org/pypi/pew/)
+## Installazione
+Per ora non esiste ancora una versione stabile di OpenFisca-Italy.
+## Iniziare a contribuire
+Per iniziare a contribuire al progetto OpenFisca-Italy leggi la seguente [linea guida](http://openfisca.org/doc/contribute/guidelines.html) che spiega nel dettaglio le modalità con le quali effettuare le operazioni. Dopodiché esegui i seguenti step:
+* Aprire un terminale; 
+* Spostarsi nella directory nella quale si vuole clonare il progetto;
+* Clonare il progetto nella directory corrente utilizzando il comando:
+```bash
+git clone https://github.com/openfisca/openfisca-italy
+```
+
+Questo è un esempio di output
+![Cmd-all](https://i.imgur.com/Lci2IVz.png)
+## Come contribuire
+Prima di poter iniziare a scrivere codice (e inziare, quindi, a definire nuove Variabili,Parametri,Riforme,Formule etc...) dovrai leggere attentamente i [concetti chiave](http://openfisca.org/doc/key-concepts.html) sui cui si basa OpenFisca (e in generale un sistema di tasse e benefici) e la [guida dalla legge al codice](http://openfisca.org/doc/coding-the-legislation/index.html).
+## Conclusione
+:tada: Complimenti, se hai letto tutte le sezioni precedenti ed hai eseguito tutte le istruzioni al loro interno sei pronto per sviluppare il sistema di tasse e benefici per l'italia.:tada:
+
+
```

Comment:
Extra lines :)
Only keep one blank line at the end of files :)

Reply:
Done :)